From 20c026db4abf9d21e5d1b56392216fb49a53ac1f Mon Sep 17 00:00:00 2001
From: David Brazdil <dbrazdil@google.com>
Date: Thu, 22 Feb 2018 17:32:33 +0000
Subject: [PATCH 142/306] Add more hidden APIs seen in use to light greylist

Found in a bugreport.

Bug: 73539503
Test: make
Change-Id: Ie8f1a1974412b86675ac1c197b20dda1736c94e4
(cherry picked from commit dac65ed99bf9e7bbdf5b0b5f294d7b0c42892d41)
---
 config/hiddenapi-light-greylist.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/hiddenapi-light-greylist.txt b/config/hiddenapi-light-greylist.txt
index dac03cdabc3..752b662e050 100644
--- a/config/hiddenapi-light-greylist.txt
+++ b/config/hiddenapi-light-greylist.txt
@@ -1691,10 +1691,12 @@ Landroid/view/accessibility/IAccessibilityManager$Stub;->asInterface(Landroid/os
 Landroid/view/accessibility/IAccessibilityManager$Stub$Proxy;-><init>(Landroid/os/IBinder;)V
 Landroid/view/ActionMode;->isUiFocusable()Z
 Landroid/view/animation/Animation;->mListener:Landroid/view/animation/Animation$AnimationListener;
+Landroid/view/Choreographer$CallbackQueue;->addCallbackLocked(JLjava/lang/Object;Ljava/lang/Object;)V
 Landroid/view/Choreographer;->doFrame(JI)V
 Landroid/view/Choreographer;->getFrameTime()J
 Landroid/view/Choreographer;->mCallbackQueues:[Landroid/view/Choreographer$CallbackQueue;
 Landroid/view/Choreographer;->mLastFrameTimeNanos:J
+Landroid/view/Choreographer;->mLock:Ljava/lang/Object;
 Landroid/view/Choreographer;->postCallback(ILjava/lang/Runnable;Ljava/lang/Object;)V
 Landroid/view/Choreographer;->removeCallbacks(ILjava/lang/Runnable;Ljava/lang/Object;)V
 Landroid/view/Choreographer;->scheduleVsyncLocked()V
-- 
2.17.1

